connectors/slack/template/storage.js

```diff
@@ -0,0 +1,51 @@
+const Sdk = require('@fusebit/add-on-sdk');
+
+var storage;
+module.exports.ensureStorage = (ctx) => {
+    if(!storage){
+        storage = Sdk.getStorageClient(ctx);
+    } 
+}
+
+module.exports.appToHandler = null;
+module.exports.tokenToApp = null;
+var mappingTimestamp;
+module.exports.get = async () => {   
+    let result = await storage.get();
+    module.exports.appToHandler = (result && result.appToHandler) || {};
+    module.exports.tokenToApp = (result && result.tokenToApp) || {};
+    mappingTimestamp = Date.now();
+}
+
+module.exports.put = async (applyChanges) => {
```

Comment:
Is it worth pulling this out into the SDK rather than special-casing it here?

Reply:
There is a trail of storage factoring things that I am tracking here fivequarters/q5#725, added this